src/lib/types.ts

```diff
@@ -1,6 +1,6 @@
 export type MapsOptions = {
     container: string;
-    coords?: mapboxgl.LngLatLike;
+    coords?: Array<mapboxgl.LngLatLike>;
```

Comment:
**Proposal**:
Split marker coordinates and coordinates for center of map into 2 different options.

Reply:
Added center option for overriding map center. Even if there are multiple map coordinates.